File: Practice/16/C++/16.cpp
//Shulzhenko
#include <iostream>
#include <string_view>

int main() {
    std::string_view str;
    std::string input;

    long long count;
    std::cin >> count;

    bool banknote_was_found = false;
    for (long long i = 0; i < count; ++i) {
        std::cin >> input;
        str = input;
        if (str.starts_with('a') && str.ends_with("55661")) {
            banknote_was_found = true;
            std::cout << str << " ";
        }
    }
    if (!banknote_was_found) {
        std::cout << -1 << std::endl;
    }
}

File: Practice/16/Python/16.py
  
n = int(input())
mon = []
for _ in range(n):
    mon = input()
    if mon.startswith('a') and mon.endswith('55661'):
        mon.append(mon)

if len(mon) == 0:
    print(-1)
else:
    print(' '.join(mon))